docker-images/openemr/Dockerfile

```diff
@@ -10,17 +10,22 @@ RUN git clone https://github.com/openemr/openemr.git --branch rel-500 --depth 1
     && chmod 666 openemr/interface/modules/zend_modules/config/application.config.php \
     && chown -R apache openemr/ \
     && mv openemr /var/www/localhost/htdocs/
+WORKDIR /var/www/localhost/htdocs/openemr
+VOLUME [ "sites" ]
```

Comment:
Should this VOLUME command have a full path? Have you verified it's pointing where you want? The docs list the commands that WORKDIR sets the local path to, and VOLUME isn't on that list...

Reply:
I came across an issue in docker's github repo that suggested that VOLUME was relative to WORKDIR  but honestly it can't hurt to make it absolute so I might as well. I'll do that in the cleanup commit in the morning!